app/Models/V2/Projects/ProjectReport.php

```diff
@@ -439,11 +445,43 @@ public function getWorkdaysTotalAttribute(): int
         return $projectReportTotal + $sumTotals('paid') + $sumTotals('volunteer');
     }
 
+    public function getWorkdaysDirectTotalAttribute(): int
+    {
+        return $this->workdays_direct ?? 0;
+    }
+
+    public function getWorkdaysConvergenceTotalAttribute(): int
+    {
+        return $this->workdays_convergence ?? 0;
+    }
```

Comment:
I think these two are superfluous. The resource can just use `workdays_direct` and `_convergence` directly, which would be in line with how the other totals are accessed. Also, since these are using SQL `sum()`, I believe that they'll always return 0 automatically if there are no matching records.